lib/macho/load_commands.rb

```diff
@@ -1119,6 +1123,97 @@ def sdk_string
       end
     end
 
+    # A load command containing the minimum OS version on which
+    # the binary was built for its platform.
+    # Corresponds to LC_BUILD_VERSION.
+    class BuildVersionCommand < LoadCommand
+      # @return [Integer]
+      attr_reader :platform
+
+      # @return [Integer] the minimum OS version X.Y.Z packed as x16.y8.z8
+      attr_reader :minos
+
+      # @return [Integer] the SDK version X.Y.Z packed as x16.y8.z8
+      attr_reader :sdk
+
+      # @return ToolEntries tool entries
```

Comment:
Nit: `ToolEntries` -> `[ToolEntries]` for YARD

Reply:
Fixed!